cma/evolution_strategy.py

```diff
@@ -4646,4 +4666,31 @@ def update(es):
     es.objective_function_complements = [_al]
     es.augmented_lagrangian = _al
     es.best_feasible = best_feasible_solution
+
+    if post_optimization:
+        kwargs_post_opt = kwargs.copy()
+        if 'options' in kwargs_post_opt:
+            kwargs_post_opt['options']['ftarget'] = 0
+        else:
+            kwargs_post_opt['options'] = {'ftarget': 0}
+
+        _, es_post_opt = fmin2(lambda x: sum(
+            [gi ** 2 if gi > 0 else 0 for gi in g(x)]) + sum(
+            [hi ** 2 if hi ** 2 > post_optimization ** 2 else 0 for hi in h(x)]),
+                               es.result.xfavorite, sigma0 / 1000, **kwargs_post_opt)
```

Comment:
After reconsideration, `sigma0 / 1000` looks too arbitrary. We should use `es.sigma` instead.